src/com/github/korniloval/matlab/editor/smartEnter/MatlabSmartEnter.kt

```diff
@@ -0,0 +1,19 @@
+package com.github.korniloval.matlab.editor.smartEnter
+
+import com.intellij.codeInsight.editorActions.smartEnter.SmartEnterProcessor
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiFile
+
+class MatlabSmartEnter : SmartEnterProcessor() {
+
+  private val processors: List<SmartEnterProcessor> =
+      listOf(
+          StatementCompletionProcessor()
+      )
+
+  override fun process(project: Project, editor: Editor, psiFile: PsiFile): Boolean =
+      processors.map { processor ->
+        processor.process(project, editor, psiFile)
+      }.reduce { acc, b -> acc && b }
```

Comment:
Should it be `||` operation instead? We may have independent processors, e.g. one that inserts `;` and one that inserts `end`